File: Codeforces/Round/0-999/654/div2/b.py
for _tes in range(int(input())) : 
    n, r = map(int, input().split())
    if n<=1 : print(1)
    else : 
        ans=int(0)
        if n<=r : 
            r=n-1
            ans=1
        print(int(ans+((r+1)*r)/2))

File: Codeforces/Round/0-999/654/div2/b.kt
fun main() { 
    repeat(readLine()!!.toInt()) { 
        var (n, x)=readLine()!!.split(" ").map{it.toLong()}
        if(n<=1) println(1)
        else { 
            var ans:Long=1; ans-=1
            if(n<=x) x=n-1; ans=1
            println(ans+((x+1)*x)/2)
        } 
    } 
} 
